example/lib/featues/io_save_image.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:image_gallery_saver/image_gallery_saver.dart';
+import 'package:pretty_qr_code/pretty_qr_code.dart';
+
+extension PrettyQrImageExtension on QrImage {
+  /// Whether or not this [QrImage], can be saved as file.
+  bool get supportsSaving => true;
```

Comment:
If we support all platforms now, it doesn't make sense.

Reply:
deleted